File: library/src/main/java/org/angmarch/views/PopUpTextAlignment.java
package org.angmarch.views;

enum PopUpTextAlignment {
    START(0),
    END(1),
    CENTER(2);

    private final int id;

    PopUpTextAlignment(int id) {
        this.id = id;
    }

    static PopUpTextAlignment fromId(int id) {
        for (PopUpTextAlignment value : values()) {
            if (value.id == id) return value;
        }
        return CENTER;
    }
}

File: library/src/main/java/org/angmarch/views/PopUpTextAlignment.kt
package org.angmarch.views

internal enum class PopUpTextAlignment(private val id: Int) {
    START(0), END(1), CENTER(2);

    companion object {
        fun fromId(id: Int): PopUpTextAlignment {
            for (value in values()) {
                if (value.id == id) return value
            }
            return CENTER
        }
    }
}